partnerships/overview.md

```diff
@@ -1,57 +1,47 @@
+2. **Community Success**: Deliver value to all communities in a consistent, repeatable, scalable way so that partners see the return on their ongoing investment and collaboration with 2i2c.
+   
+## Major areas of responsibility
 
-## Represent this strategy internally and externally
+Partnerships defines strategy for the following areas and aligns it with the overall strategy of 2i2c.
 
-- Amplify [Voice of Partner](https://www.zs.com/insights/high-techs-new-challenge-hearing-voices-of-channel-partners) for 2i2c stakeholders in internal discussions (e.g. with 2i2c's engineering and support teams).
-- Collaborate with Product and Community Lead to represent Sustainability strategy in product/service decisions.
-- Communicate 2i2c's sustainability strategy and model to external stakeholders
+### Sustainability
 
-## Design and develop new revenue mechanisms
+Establish, support and renew partnerships so that 2i2c has the financial resources to advance on its mission.
 
-- Establish and sustain revenue-generating relationships with partner organizations
-- Establish ongoing relationships with cloud vendors or resellers
-- Collaborate with product and engineering teams.
+- **Marketing**: Describe 2i2c’s value proposition externally and attract new leads.
+- **Sales**: Generate new and renew agreements and service contracts.
+- **Grants**: Identify and apply for grant opportunities that align with 2i2c’s service strategy
+- **Invoicing and cost recovery**: Ensure 2i2c recovers the cost that it is due via contracts.
+- **Pricing and contract strategy**: Set pricing and agreement terms for partners so that paid services meet our growth goals.
+- **Market research and competitive analysis**: Understand 2i2c’s value proposition and business model relative to other services and products.
+- **Business aspects of joint ventures**: Understand the financial and strategic impact of partnerships with other services and products (e.g. with cloud vendors, cloud resellers, publishers, technology companies).
 
-## Identify, develop, and pursue funding opportunities
+### Community Success
 
-For example:
+Deliver value to all communities in a consistent, repeatable, scalable way so that partners see the return on their ongoing investment and collaboration with 2i2c. 
 
-- Partnerships that grow our capacity
-- Grants for the organization or for deliverables
-- Philanthropy and giving
+- **Onboarding**: Set expectations and provide guidance for new communities that are using our service.
+- **Training**: Help communities understand how to use our products and services to achieve value.
+- **Community satisfaction**: Understand the ongoing satisfaction that communities have with the service and identify ways to improve their experience.
+- **Account management**: Establish and sustain relationships with partner organizations to ensure they are successful.
+- **Grant management**: Ensure that the goals and deliverables of our grants are understood and met.
```

Comment:
This feels like "Project management" to me. Is that your intention? E.g., if we hired a project manager would you imagine them reporting somewhere in the Partnerships chain?

We can have it here for now and revisit later if it becomes relevant.

Reply:
These headings were introduced to describe major functional areas or work streams under Partnerships. I do not believe Partnerships is the natural umbrella for all "Project Manager" roles within 2i2c. Development projects mostly involving engineering (e.g. SOLyR?) should not be managed by a role in Partnerships.

Reply:
I think we're aligned on this - though there's an interesting question about who is the ultimate "owner" of a grant and its deliverables in 2i2c, given that most grants are cross-functional. I think this is a role that we're missing and as a result I am the de-facto owner of all grants which is not sustainable.